salt/core/firewall/files/firewall_reset.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#
+# This file is managed by Dogsbody Technology Ltd.
+#   https://www.dogsbody.com/
+#
+# Description:  A script to reset iptables rules.
+#
+# Usage:  $0
+#
+
+iptables -P INPUT ACCEPT
+iptables -P FORWARD ACCEPT
+iptables -P OUTPUT ACCEPT
+iptables -F
+iptables -X
+iptables -Z
+iptables -t nat -F
+iptables -t nat -X
+iptables -t mangle -F
+iptables -t mangle -X
+iptables -t raw -F
+iptables -t raw -X
+ip6tables -P INPUT ACCEPT
+ip6tables -P FORWARD ACCEPT
+ip6tables -P OUTPUT ACCEPT
+ip6tables -F
+ip6tables -X
+ip6tables -Z
+ip6tables -t nat -F
```

Comment:
I noticed that `firewall.sh` doesn't reset nat for ipv6. Should that be added?

Reply:
Added in new commit.